assets/airports/egcc.json

```diff
@@ -1,890 +1,3153 @@
+            "rwy": {
+                "EGCC23L": []
+            },
+            "body": ["_MCT2313.2", ["_MCT254008", "A25+|A50-"], ["_MCT254013", "A40+|A50-"], "^_MCT254015", ["_POL219036", "A50"]],
+            "exitPoints": {
+                "MONTY": ["MONTY"]
+            },
+            "draw": [
+                ["_MCT2313.2", "_MCT254008", "_MCT254013", "_MCT254015", "_POL219036", "MONTY"]
+            ]
+        },
+        "MONTY1R": {
+            "icao": "MONTY1R",
+            "name": "Monty One Romeo",
+            "rwy": {
+                "EGCC23R": []
+            },
+            "body": ["_MCT231003", ["_MCT254008", "A25+|A50-"], ["_MCT254013", "A40+|A50-"], "^_MCT254015", ["_POL219036", "A50"]],
+            "exitPoints": {
+                "MONTY": ["MONTY"]
+            },
+            "draw": [
+                ["_MCT231003", "_MCT254008", "_MCT254013", "_MCT254015", "_POL219036", "MONTY"]
+            ]
+        },
+        "POL4S": {
+            "icao": "POL4S",
+            "name": "Pole Hill Four Sierra",
+            "rwy": {
+                "EGCC05L": []
+            },
+            "body": ["_RWY23R", "_MCT053002", "_MCT053007", ["_POL182012", "A40+|A50-"], ["_POL182008", "A50"]],
+            "exitPoints": {
+                "POL": ["POL"]
+            },
+            "draw": [
+                ["_RWY23R", "_MCT053002", "_MCT053007", "_POL182012", "_POL182008", "POL"]
+            ]
+        },
+        "POL1Z": {
+            "icao": "POL1Z",
+            "name": "Pole Hill One Zulu",
+            "rwy": {
+                "EGCC05R": []
+            },
+            "body": ["_RWY23L", "_MCT053002", "_MCT053007", ["_POL182012", "A40+|A50-"], ["_POL182008", "A50"]],
+            "exitPoints": {
+                "POL": ["POL"]
+            },
+            "draw": [
+                ["_RWY23L", "_MCT053002", "_MCT053007", "_POL182012", "_POL182008", "POL"]
+            ]
+        },
+        "POL1Y": {
+            "icao": "POL1Y",
+            "name": "Pole Hill One Yankee",
+            "rwy": {
+                "EGCC23L": []
+            },
+            "body": ["^_MCT2313.2", ["_MCT294007", "A25+|A50-"], "_MCT302008", "XUMAT", ["_POL221016", "A40+|A50-"], ["_POL221009", "A50"]],
+            "exitPoints": {
+                "POL": ["POL"]
+            },
+            "draw": [
+                ["_MCT2313.2", "_MCT231006", "_MCT294007", "_MCT302008", "XUMAT", "_POL221016", "_POL221009", "POL"]
+            ]
+        },
+        "POL5R": {
+            "icao": "POL5R",
+            "name": "Pole Hill Five Romeo",
+            "rwy": {
+                "EGCC23R": []
+            },
+            "body": ["^_MCT231003", ["_MCT294007", "A25+|A50-"], "_MCT302008", "XUMAT", ["_POL221016", "A40+|A50-"], ["_POL221009", "A50"]],
+            "exitPoints": {
+                "POL": ["POL"]
+            },
+            "draw": [
+                ["_MCT231003", "_MCT231006", "_MCT294007", "_MCT302008", "XUMAT", "_PO
```

Comment:
Departing spawnPatterns will need to be extended outside the TMA

Reply:
I added the airways for them but didn't put them in the spawnPatterns 🤦‍♂️